doc/tests/block_name.rst

```diff
@@ -0,0 +1,13 @@
+``block name``
+==============
+
+.. versionadded:: 1.28
+    The ``block name`` test was added in Twig 1.28.
+
+``block name`` checks if a block is defined in the current context of the template.
+
+.. code-block:: jinja
+
+    {% if 'title' is block name %}
```

Comment:
I still find this very weird to read and not easy to understand. I don't have any better proposal for now.